src/lsp/superbol_free_lib/vscode_extension.ml

```diff
@@ -583,13 +583,26 @@ let contributes =
         ~command:"superbol.coverage.reload"
         ~title:"Update Coverage"
         ~category:"SuperBOL";
+      Manifest.command ()
+        ~command:"superbol.cfg.open"
+        ~title:"Open a CFG of the current program"
+        ~category:"SuperBOL";
+      Manifest.command ()
+        ~command:"superbol.cfg.open.arc"
+        ~title:"Open an arc CFG of the current program"
```

Comment:
"Arc CFG"? Instead I think "Show control-flow as an arc-diagram" may fit better: shorter, straight to the point. (Same above with the prefix of this sentence).